### PR TITLE
Seek Bars track time and volume

### PR DESCRIPTION
**Pull request by timschmidtdev**
@droberts84
Tracking the time of the currently playing song and volume worked great!

### CODE REVIEW

File: app/templates/player_bar.html [hunk 1]
```diff
@@ -22,15 +22,15 @@
...
+                <div class="current-time">{{ playerBar.songPlayer.currentTime }}</div>
                 <div class="total-time">{{ playerBar.songPlayer.currentSong.duration }}</div>
             </div>
             <h2 class="artist-song-mobile">{{ playerBar.songPlayer.currentSong.title }}</h2>
             <h3 class="artist-name">{{ playerBar.albumData.artist }}</h3>
         </div>
         <div class="control-group volume">
             <span class="ion-volume-high icon"></span>
-            <seek-bar></seek-bar>
+            <seek-bar value="{{ playerBar.songPlayer.volume }}" max="100" on-change="playerBar.songPlayer.setVolume(value)"></seek-bar>
```
Comment (drobtravels) [inline app/templates/player_bar.html:33]: What would cause `playerBar.songPlayer.volume` to change?  Does it ever change?

Comment (timschmidtdev) [reply app/templates/player_bar.html:33]: When the user moves the volume thumb is the only way to update the value of Song.Player.volume.

Comment (drobtravels) [reply app/templates/player_bar.html:33]: When the user moves the thumb, it will call the `on-change` function.  Where does volume get changed?

Comment (timschmidtdev) [reply app/templates/player_bar.html:33]: Weird. I removed the entire value and the volume thumb still worked in the view. The assignment in Bloc mentioned to add a value and a max to seek-bar. Let's look at this during our meeting tomorrow.

**Pull request merged**